### DISCUSSION TITLE
I cannot publish the site

### DISCUSSION THREAD

**Discussion by naberna** [swellstores/community#368] [Help] [answered]
Hello, 

I have a site that I should migrate but I found the migration progress hard. What I would like to do is just publish the site to production but I got errors. Here is what is in my log:
Failed to publish just-check--dev => just-check

Can you help me with this or is there any easy way for migration?

**Comment by logeshswell** [comment, marked as answer]
@naberna The deployment issue is fixed in your store and you should be able to publish the storefront now.
Please check and confirm.

**Comment by naberna** [comment]
@logeshswell thank you very much for your fix. Unfortunately, I have one more problem in regards to Paypal, I cannot connect my account to Paypal. Can you please have a look at that also?

**Reply by logeshswell** [reply]
What is the error do you get?
Can you share the screenshot or screen recording of the issue so that I can assist you better.

**Reply by naberna** [reply]
Hello, as you can see I enabled Paypal but even I click, it doesnt do anything.

**Reply by logeshswell** [reply]
Can you try again now? Looks like the button is not responding rarely sometimes.

I tried hard refreshing the page and it is redirecting to Paypal page now.

**Reply by naberna** [reply]
Thank you very much for your reply. Can you see our live website? I still see the problem exists.

Kindly,

**Reply by naberna** [reply]
Dear,

Is it possible for you to share the code with me so I can have a look? It is quite urgent!

**Reply by logeshswell** [reply]
@naberna I see that you enabled Paypal Commerce under Payment settings. What other issue do you face?
BTW, which browser do you use?

**Reply by naberna** [reply]
I use Chrome.

In the site, I add a product to my basket. I go to here: https://www.just-check.co.uk/checkout/712c728fb736d65fe6f46b712fabbced/shipping and then I click Paypal to log myself in and pay. I click Paypal, and it looks like opening the Paypal Modal, but the modal closes immediately.

**Reply by logeshswell** [reply]
I see the below error is coming from Paypal as follows:
```json
{
   "name":"UNPROCESSABLE_ENTITY",
   "details":[
      {
         "issue":"NOT_ENABLED_TO_VAULT_PAYMENT_SOURCE",
         "description":"The API caller or the merchant on whose behalf the API call is initiated is not allowed to vault the given source. Please contact PayPal customer support for assistance."
      }
   ],
   "message":"The requested action could not be performed, semantically incorrect, or failed business validation.",
   "debug_id":"ca8adf54f085d",
   "links":[
      {
         "href":"https://developer.paypal.com/docs/api/orders/v2/#error-NOT_ENABLED_TO_VAULT_PAYMENT_SOURCE",
         "rel":"information_link",
         "method":"GET"
      }
   ]
}
```
There should be some configuration disabled in your Paypal account. Can you check with Paypal support regarding the same and keep us posted?

**Reply by naberna** [reply]
Dear, thanks for your advice I could solve the problem with PayPal. But currently, I am having a problem with amazon. Even if I make the button enabled, it is not visible on the site. Can you please have a look?
